### PR TITLE
Hotfix/fixed null on config urls

### PR DESCRIPTION
**Pull request by AaronNeonDigital**
-fixed default url generation set to null as docs states

#266

### CODE REVIEW

File: docs/src/getcandy/products.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ If you decide to delete an attribute, this will cause the association to be drop
 :::
 
 ### Retrieving the product type relationship
-If you have a product, you can fetch it's product like like so:
+If you have a product, you can fetch its product type like like so:
```
Comment (markmead) [inline docs/src/getcandy/products.md:88]: Says `like like so`

Comment (glennjacobs) [reply docs/src/getcandy/products.md:88]: @AaronNeonDigital can you fix that up?

**Comment by glennjacobs** [comment]
@alecritson do we want it set as NULL by default like the docs say? Or should we change the docs?

https://docs.getcandy.io/getcandy/urls.html#automatically-generating-urls

**Comment by alecritson** [comment]
@glennjacobs I'm more on the side of changing the docs, since changing code will change how it currently works so would need something in the upgrade guide. Unless we want the functionality to change.

**Comment by glennjacobs** [comment]
> 

@alecritson It's not code as such, just the default config value.

**Comment by glennjacobs** [comment]
@alecritson gotcha. Let's update the docs then. @AaronNeonDigital did you want to attempt this, or shall we?

**Comment by glennjacobs** [comment]
@alecritson you happy with this now?

**Comment by alecritson** [comment]
@glennjacobs Yeah I think it's fine!

**Review by glennjacobs — APPROVED** [review]

**Pull request merged**